### DISCUSSION TITLE
dependency of p-editor is deprecated

### DISCUSSION THREAD

**Discussion by dhananjay19100** [primefaces/community#1300] [PrimeNG] [answered]
![Screenshot from 2024-02-27 11-47-37](https://github.com/primefaces/community/assets/91648883/ac4b9c38-3a96-4f82-af14-d93285e8e363)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for reporting the issue we'll try to address this in v17.

**Reply by dhananjay19100** [reply]
but i am using v15
@cetincakiroglu

**Reply by cetincakiroglu** [reply]
Unfortunately, we cannot provide a fix for v15 because it's out of support unless you don't have LTS access, and v15-lts will be out of support when the Angular v18 comes out. 

Also, last update date of Quill was 4 years ago, the issue is related to Quill's internal methods, probably they were using something valid at the time but became deprecated within 4 years. As PrimeNG team we cannot provide a fix for Quill's internal methods. It's an outdated library, we've plans to replace it in the future. https://www.npmjs.com/package/quill

The best option, for now, is to use one of the other open-source editors that receives updates, unlike Quill.

**Reply by ThoSap** [reply]
As you can see Quill is again under active development since September 2023.
The new v2.0 version, which will be released soon no longer has the mutation events which will be removed from Chrome/Chromium on July 2024 (Chromium 127).
https://github.com/quilljs/quill/issues/3806#issuecomment-1933250524
https://github.com/quilljs/quill/releases

I do not think the community will switch to TinyMCE, because for open-source it is limited to 1,000 editor loads per month and the CKEditor has many features behind a paywall, for example paste from Microsoft Word, which works in Quill v2.0.

**Reply by ThoSap** [reply]
See https://github.com/primefaces/primeng/issues/14721

**Reply by ThoSap** [reply]
@dhananjay19100 this is fixed with 🥳🥳🥳
https://github.com/primefaces/primeng/releases/tag/17.18.3

Just update to this version and Quill 2.0.2
